presto-accumulo-iterators/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>334-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-accumulo-iterators</artifactId>
+    <description>Accumulo Iterators for the Presto - Accumulo Connector</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.accumulo.version>1.7.4</dep.accumulo.version>
```

Comment:
Since we want this to match the Accumulo connector, let's move this to the root POM, and add a property for the Accumulo Hadoop version:
```xml
<dep.accumulo.version>1.7.4</dep.accumulo.version>
<dep.accumulo-hadoop.version>2.7.7-1</dep.accumulo.version>